grpc/agent_service_test.cc

```diff
@@ -31,6 +32,7 @@
 #include <mujoco/mujoco.h>
 #include "grpc/agent.grpc.pb.h"
 #include "grpc/agent.pb.h"
+#include "third_party/mujoco_mpc/grpc/agent.proto.h"
```

Comment:
this is wrong.  I'll go fix the underlying cause, but can you please fix this here?

Reply:
I fixed it in https://github.com/deepmind/mujoco_mpc/commit/7c9c7a6c7766042a9107a95f07d7d09a5212a0b9
Please update the PR and check that it's fixed.

Reply:
Can you update the PR and make sure it goes away?  Supposedly I fixed it in https://github.com/deepmind/mujoco_mpc/commit/7c9c7a6c7766042a9107a95f07d7d09a5212a0b9

Reply:
Fixed. 

There is a similar "third_party" issue in interface.cc that I fixed too.